### ISSUE TITLE
DNSRecord/DNSZone/Nameserver are not tied to AWSAccount

### ISSUE THREAD

**Issue by marco-lancini** [lyft/cartography#701]
The AWS resources of [DNSRecord](https://github.com/lyft/cartography/blob/master/docs/schema/aws.md#dnsrecord), [DNSZone](https://github.com/lyft/cartography/blob/master/docs/schema/aws.md#dnszone), and [Nameserver](https://github.com/lyft/cartography/blob/master/docs/schema/aws.md#nameserver) are not tied to an AWSAccount (i.e., the relationship `(AWSAccount)-[RESOURCE]->(x)` is missing).

This deviates from the standard convention (since all other resources can be accessed by an `AWSAccount`), and it prevents from associating these resources to the account they reside in

**Comment by achantavy** [comment]
I agree this should be fixed. Our new data model has a convention where all nodes will tie back to their sub resource, so once we refactor it all then this issue will be fixed.

**Comment by achantavy** [comment]
DNS ones addressed in https://github.com/cartography-cncf/cartography/pull/1703.

Still waiting for the AWSPolicy one to be addressed in https://github.com/cartography-cncf/cartography/pull/1710
